src/app/shared/actions/account-actions/actions/account-delete-action.ts

```diff
@@ -1,53 +1,14 @@
 import { Injectable } from '@angular/core';
-import { Observable } from 'rxjs/Observable';
-import { JobsNotificationService } from '../../../services/jobs-notification.service';
 import { BaseAccountAction } from './base-account-action';
-import { DialogService } from '../../../../dialog/dialog-service/dialog.service';
-import { AccountService } from '../../../services/account.service';
-import { Account } from '../../../models/account.model';
 
 
 @Injectable()
-export class AccountDeleteAction extends BaseAccountAction {
+export class AccountDeleteAction implements BaseAccountAction {
```

Comment:
Do we need Actions to be injectable? we used that before because each action had services injected inside, now we moved all the logic to effects. let's remove that. maybe we could have all the actions just a constants with the same interface in one file?